src/datasources/test-plans/components/TestPlansQueryEditor.tsx

```diff
@@ -61,5 +89,7 @@ export function TestPlansQueryEditor({ query, onChange, onRunQuery, datasource }
 
 const tooltips = {
   outputType: 'This field specifies the output type to fetch test plan properties or total count.',
-  properties: "Specifies the properties to be queried."
+  properties: "Specifies the properties to be queried.",
+  orderBy: "Specifies the field to order the queried test plans by.",
+  descending: "Specifies whether to return the test plans in descending order."
```

Comment:
```suggestion
  orderBy: "Specifies the field to order the queried test plans by.",
  descending: "Specifies whether to return the test plans in descending order."
```
![image](https://github.com/user-attachments/assets/6ab3ff0b-a237-4772-bf99-ad1bd8b15454)
![image](https://github.com/user-attachments/assets/7cbfa27d-ed55-4ee1-a664-a2aacff67c92)
![image](https://github.com/user-attachments/assets/c3516226-892d-4fe5-b1f1-57446ca94a37)

@WillCawley  Could you please review user visible strings?

Reply:
@priya-ni Here are some suggestions:
- orderBy: “This field specifies the query order of the test plans.”
- descending: “This toggle returns the test plans query in descending order.”
- Text Field: “Select a field to set the query order”

Reply:
Applied the suggestion.